pivorakmeetup/pivorak-web-app
# Add search to the index talks page

### Description

there is no search bar on the pivorak.com/talks page, it should be added to have navigation through talks faster.

a search bar has been added to the pivorak.com/talks page, test steps and changes are described below.

### How to test instructions

1. go to /talks page
2. verify that search bar shown up on the page (see attachments)
3. put some title of the talks and press search button
4. talks with appropriated title or description should be shown up on the page.

### Pre-merge checklist
 
- [ improvement] The PR relates to a single subject with a clear title and description
- [ +] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

| Before                                      | After                                       |
| ------------------------------------------- | ------------------------------------------- |
| 
![before_talks](https://user-images.githubusercontent.com/10061671/28302479-393e4fc6-6b96-11e7-9d05-71472acf099a.png) | ![after_search](https://user-images.githubusercontent.com/10061671/28302485-4098d7aa-6b96-11e7-96a7-e656707f195b.png)
 |
